pmd-plsql/etc/grammar/PLSQL.jjt

```diff
@@ -4984,6 +4984,7 @@ TOKEN [IGNORE_CASE]:
 <CONSTRAINT: "CONSTRAINT"> |
 <CONSTRUCTOR: "CONSTRUCTOR"> |
 <CONTINUE: "CONTINUE"> |
+<CONVERSION: "CONVERSION"> |
```

Comment:
we need to be careful - this is neither a reserved word nor a keyword and can be used as an identifier in other contexts.